### ISSUE TITLE
Intermittently losing text from the menu bar

### ISSUE THREAD

**Issue by IanBlakeley** [linuxmint/cinnamon#9953]
```
 * Version = Cinnamon 4.8.6
 * Distribution = Mint 20.1
 * Graphics = i5 on board graphics
 * 64 bit
 ```
Randomly usually when I've gone away from the PC for a bit and it has been running for a few hours the text on the menu bar goes missing and I end up with.

![missing txt](https://user-images.githubusercontent.com/36182904/108958490-07ab8380-76ae-11eb-906e-61658b485575.jpg)

Restarting cinnamon and it comes back

![cinnamon restarted](https://user-images.githubusercontent.com/36182904/108958662-3fb2c680-76ae-11eb-86ee-c7ea3cd6d8c1.jpg)

Long story short it seems to be an interaction with the blink speed selected in an app birdtray reducing the blink speed resolves the issue for me. Reporting here at their suggestion [https://github.com/gyunaev/birdtray/issues/425#issuecomment-783839871](url)

**Steps to reproduce**
Increase icon blink rate to around 75%, at 50% I have not seen the issue

`
